### DISCUSSION TITLE
请教：快捷键扩展如何使用？

### DISCUSSION THREAD

**Discussion by i110077** [the1812/Bilibili-Evolved#4015] [提问 / Q&A]
我安装了快捷键扩展 - 视频倍速的插件，请问如何使用啊？只能看到安装了这个插件，找不到设置的地方，也无法查看快捷键是什么
![Screenshot 2023-02-20 at 15-32-16 【卡片盒笔记法】碎片化时代里解决你的知识焦虑，使用卡片笔记写作法来做知识管理，卢曼的卡片盒，Zettelkasten 。_哔哩哔哩_bilibili](https://user-images.githubusercontent.com/83385874/220041905-426d6000-0ce4-45db-b35c-20ab437177fc.png)

**Comment by the1812** [comment]
还要装一下 `快捷键扩展` 本身

**Reply by Geeyun-JY3** [reply]
要不插件的简介都提一嘴要先安装哪个组件吧……
